cv32e40s/tests/cfg/pmp.yaml

```diff
@@ -0,0 +1,10 @@
+name: pmp
+description: Default pmp configuration for CV32E40S simulations
+compile_flags: >
+   +define+PMP_ENABLE_64
+ovpsim: >
+    --override root/cpu/PMP_registers=64
+    --showoverrides
```

Comment:
The --showoverrides line should be commented out or removed before merge

Reply:
Removed the --showoverrides line